tests/test_tank.py

```diff
@@ -190,6 +189,10 @@ def test_mass_based_tank():
         discretize=None,
     )
 
+    # Assert volume bounds
+    assert (real_tank_lox.gas_height <= real_tank_lox.geometry.total_volume).all
```

Comment:
This line is wrong, it should be 


```suggestion
    assert (real_tank_lox.gas_height <= real_tank_lox.geometry.top).all
```